app/ChatPanelHeader.tsx

```diff
@@ -108,14 +108,50 @@ function ChatPanelHeader() {
         <ThreadActionsMenu
           thread={{ id: currentThreadId, name: currentThreadName }}
         >
-          <Button variant="ghost" size="sm" mr="auto">
-            {currentThreadName}
-            <CaretDownIcon />
+          <Button
+            variant="ghost"
+            size="sm"
+            flex="1"
+            minW={0}
+            justifyContent="flex-start"
+          >
+            <Flex align="center" gap={1} w="100%" minW={0}>
+              <Tooltip content={currentThreadName} showArrow>
+                <Text
+                  as="span"
+                  flex="1"
```

Comment:
```suggestion
            flexShrink="1"
            mr="auto
```
needed to allow for space between the header items; keeps thread name button to the left